edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/Program.cs

```diff
@@ -255,7 +255,7 @@ public static async Task<int> MainAsync(IConfiguration configuration)
                 {
                     case "twin":
                         bool enableStreams = configuration.GetValue(Constants.EnableStreams, false);
-                        int requestTimeoutSecs = configuration.GetValue(Constants.RequestTimeoutSecs, 600);
+                        int requestTimeoutSecs = configuration.GetValue(Constants.RequestTimeoutSecs, 30);
```

Comment:
This is the part that I'm not super sure about if I should change the default request timeout from 600s to 30s for all the default. I don't really have a clean way to set a particular `GetModuleLogs` DM's default request timeout to be 30s. Any suggestions?

Reply:
dm call has a timeout that can be set to ResponseTimeout in MethodRequest class. The default change is a big difference, it might cause issues with slower connections I assume.